tools/creds/slack_users_map.yaml

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
Perhaps we need to sort the file ?

Reply:
@liangxia not necessary, I create the file and add the user as they join at the moment.